Models/Images/Image.cs

```diff
@@ -1,35 +1,16 @@
-﻿namespace Models.Images
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Models.Images
 {
     public class Image
     {
+        [Required]
         public int Id { get; set; }
+        [Required]
         public int CarId { get; set; }
+        [Required]
         public long Timestamp { get; set; }
         public string Path { get; set; }
```

Comment:
Storing image as bytes in system is a very bad approach, leave Path only and make it required